doc/api/child_process.md

```diff
@@ -266,6 +266,9 @@ changes:
     normally be created on Windows systems. **Default:** `false`.
   * `windowsVerbatimArguments` {boolean} No quoting or escaping of arguments is
     done on Windows. Ignored on Unix. **Default:** `false`.
+  * `shell` {string} Shell to execute the command with.
+    **Default:** `'/bin/sh'` on UNIX, `process.env.ComSpec` on Windows. See
+    [Shell Requirements][] and [Default Windows Shell][].
```

Comment:
There are several places in this file where it is stated that a shell is not spawned. These will have to be updated otherwise the docs will be contradictory.

Reply:
thanks for the review, i will take care of this today